tests/cover/test_renaming.py

```diff
@@ -22,7 +22,7 @@
 
 def test_can_rename_arguments_in_a_function_with_no_docstring():
     @renamed_arguments(old_arg='new_arg')
-    def f(old_arg, new_arg):
+    def f(new_arg, old_arg):
```

Comment:
👍

Reply:
Oh, right, sorry. These are failing tests because in the real system the arguments all have defaults.

...

Eight time lucky? 😢

Reply:
you don't like to make things easy, huh? 😉

Reply:
I thought I was! This all made much more sense in my head when I wrote it.

Reply:
(And this is why we have a much more nitpicky review process for public APIs than internal ones...)